raiden/api/v1/resources.py

```diff
@@ -209,3 +209,15 @@ def put(self, token_address, funds, initial_channel_target, joinable_funds_targe
 
     def delete(self, token_address):
         return self.rest_api.leave(token_address=token_address)
+
+    def get(self, token_address):
+        return self.rest_api.get_connection_manager(token_address=token_address)
+
+
+class ConnectionManagersResource(BaseResource):
+
+    def __init__(self, **kwargs):
+        super(ConnectionManagersResource, self).__init__(**kwargs)
```

Comment:
Noticed a lot of these `noop` constructors in the resources classes. Any special reason for them to exist?